File: Chapter 1/TestPerson.cs
using System;

namespace ConsoleApplication1
{
    class TestPerson
    {
        static void Main(string[] args)
        {

            Person joe = new Person();

            joe.Name = "joe";

            Console.WriteLine(joe.Name);

            Console.ReadLine();
        }
    }

    public class Person
    {

        //Attributes  
        private String _Name;
        private String _Address;

        //Methods  
        public String Name
        {
            get { return _Name; }
            set { _Name = value; }
        }


        public String Address
        {
            get { return _Address; }
            set { _Address = value; }
        }

    }
}

File: Chapter 1/TestPerson.vb
Module TestPerson

    Sub Main()

        Dim joe As Person = New Person

        joe.Name = "joe"

        Console.WriteLine(joe.Name)

        Console.ReadLine()

    End Sub

End Module

Public Class Person

    Private _Name As String
    Private _Address As String

    Public Property Name() As String
        Get
            Return _Name
        End Get
        Set(ByVal value As String)
            _Name = value
        End Set
    End Property

    Public Property Address() As String
        Get
            Return _Address
        End Get
        Set(ByVal value As String)
            _Address = value
        End Set
    End Property

End Class

